frontend/src/components/Pages/InspectionPage/InspectionTable.tsx

```diff
@@ -40,143 +45,200 @@ interface IProps {
                     </>
                 )
+            }
+            lastCompleted = TranslateTextWithContext('Never')
+        } else {
+            status = inspection.missionDefinition.inspectionFrequency
+                ? getInspectionStatus(inspection.deadline!)
+                : TranslateTextWithContext('No planned inspection')
+            lastCompleted = formatDateString(mission.lastRun.endTime!)
         }
-        return (
-            <>
-                {GreenCircle} {TranslateText('Up to date')}
-            </>
-        )
     }
 
-    const getInspectionRow = (inspection: Inspection) => {
-        const mission = inspection.missionDefinition
-        let status
-        let lastCompleted: string = ''
-        const isScheduled = Object.keys(ongoingMissions).includes(mission.id) && ongoingMissions[mission.id]
-        if (isScheduled) {
-            status = (
-                <>
-                    {GreenCircle} {TranslateText('Already scheduled')}
-                </>
-            )
-        } else {
-            if (!mission.lastRun || !mission.lastRun.endTime) {
-                if (inspection.missionDefinition.inspectionFrequency) {
-                    status = (
-                        <>
-                            {RedCircle} {TranslateText('Not yet performed')}
-                        </>
-                    )
-                } else {
-                    status = TranslateText('No planned inspection')
-                }
-                lastCompleted = TranslateText('Never')
-            } else {
-                status = inspection.missionDefinition.inspectionFrequency
-                    ? getInspectionStatus(inspection.deadline!)
-                    : TranslateText('No planned inspection')
-                lastCompleted = formatDateString(mission.lastRun.endTime!)
-            }
-        }
+    return (
+        <Table.Row key={mission.id}>
+            <Table.Cell>{status}</Table.Cell>
+            <Table.Cell>
+                <Typography
+                    link
+                    onClick={() => navigate(`${config.FRONTEND_BASE_ROUTE}/mission-definition/${mission.id}`)}
+                >
+                    {mission.name}
+                </Typography>
+            </Table.Cell>
+            <Table.Cell>{mission.comment}</Table.Cell>
+            <Table.Cell>{mission.area.areaName}</Table.Cell>
+            <Table.Cell>{lastCompleted}</Table.Cell>
+            <Table.Cell>{inspection.deadline ? inspection.deadline.toDateString() : ''}</Table.Cell>
+            <Table.Cell>
+                <StyledIcon
+                    color={`${tokens.colors.interactive.focus.hex}`}
+                    name={Icons.AddOutlined}
+                    size={16}
+                    title={TranslateTextWithContext('Add to queue')}
+                    onClick={() => {
+                        openDialog()
+                        setScheduledMissions([mission])
+                    }}
+                />
```

Comment:
Suggest changing to a button. Currently the pointer doesn't indicate that it's clickable when hovering

Reply:
Thanks, I've changed the cursor when we enter the card. In figma it is a card so I would like to keep it that way and also I want the background to remain white and only the colors to change when you hover the queue all mission button